database/models/audit.go

```diff
@@ -0,0 +1,8 @@
+package models
+
+// Event is a model of how the data is represented in a database
+type Event struct {
```

Comment:
I think the name needs to match the table. I also feel like the table name and this struct should be called AuditEvent as it is more descriptive of the content.

Reply:
Table has been renamed to audit_event and struct AuditEvent to follow best practices of both sql/go